network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -207,6 +207,7 @@ def __str__(self):
     base_form_class = BuyersGuideProductCategoryForm
 
     search_fields = [
+        index.SearchField("name"),
```

Comment:
I'm a little confused here. My understanding after reading [Wagtail doc](https://docs.wagtail.org/en/stable/releases/5.0.html#elasticsearch-backend-no-longer-performs-partial-matching-on-search), it seems like having `AutocompleteField` is enough?

Reply:
That's what I thought as well, but then on the [next point release (5.1)](https://docs.wagtail.org/en/stable/releases/5.1.html#search-within-chooser-interfaces-requires-autocompletefield-for-full-functionality) they changed it so that we actually need both to preserve the previous functionality